Example/CardParts/CardPartBarViewCardController.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  CardPartBarViewCardController.swift
+//  CardParts_Example
+//
+//  Created by Roossin, Chase on 5/23/18.
+//  Copyright © 2018 CocoaPods. All rights reserved.
```

Comment:
Did you mean to have these marked CocoaPods?

Reply:
Oops :D